schema_based_on_diagram.sql

```diff
@@ -65,6 +65,18 @@ CREATE INDEX ON invoice_items (treatment_id);
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/229867981-08afa9c1-c4fb-4a31-9569-3553a5900862.png)

**[OPTIONAL]**

It would be very great if you make all your commit messages more meaningful and capitalize because I saw your commit is not capitalized and meaningful enough 😨 Kindly consider them making capitalized and meaningful to make your pull request professional 👍